inc/core/templates/theme-options-page.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * Theme admin functions.
+ *
+ * @package BCGov Block Theme
+ */
+
+namespace Bcgov\Theme\Block;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit( '<h1>Direct access denied.</h1>' );
```

Comment:
Don't know if it's necessary for us to present a multi-lingual experience to those attempting direct access but:

```suggestion
        $exit_msg = esc_html_e('Direct access denied.');
	exit( '<h1>' . $exit_msg . '</h1>' );
```

Reply:
Added: used the return version rather than echo.

Reply:
Valid point. Removed translation functions on these pages. Decreases maintenance complexity.